packages/react-docs/RELEASE-NOTES.md

```diff
@@ -4,6 +4,114 @@ section: get-started
+  - Fixed handleMainClick function signature ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Progress:** 
+  - Added warning variant ([#4834](https://github.com/patternfly/patternfly-react/pull/4834))
+- **Progress:** 
+  - Added truncation ([#4842](https://github.com/patternfly/patternfly-react/pull/4842))
+- **Select:** 
+  - Added support for divider in select ([#4772](https://github.com/patternfly/patternfly-react/pull/4772))
+  - Cleaned up some typeahead behaviors ([#4864](https://github.com/patternfly/patternfly-react/pull/4864))
+  - Fixed grouped Select generating invalid HTML ([#4919](https://github.com/patternfly/patternfly-react/pull/4919))
+  - Added support to pass chip group props on typeaheadmulti variant ([#4855](https://github.com/patternfly/patternfly-react/pull/4855))
+  - Updated to return the string value for select chips ([#4915](https://github.com/patternfly/patternfly-react/pull/4915))
+  - Passed aria-label / aria-labelledby to the `<ul role="listbox">` to prevent axe issues ([#4931](https://github.com/patternfly/patternfly-react/pull/4931))
+  - Updated to remove the clear button button when onClear function is not specified for typeahead ([#4887](https://github.com/patternfly/patternfly-react/pull/4887))
+- **Tabs:** 
+  - Added support for light scheme variant in Tabs && TabContent ([#4832](https://github.com/patternfly/patternfly-react/pull/4832))
+  - Conditionally render aria-controls when mountOnEnter ([#4796](https://github.com/patternfly/patternfly-react/pull/4796))
+- **Tooltip** 
+  - Updated to export TooltipProps as a type to help babel ([#4854](https://github.com/patternfly/patternfly-react/pull/4854))
+
+### Catalog view extension:** 
+  - Added back catalogue items ([#4940](https://github.com/patternfly/patternfly-react/pull/4940))
+
+### Charts
+- Replaced or used exact lodash imports ([#4830](https://github.com/patternfly/patternfly-react/pull/4830))
+
+### Icons
+- Used lying cast ([#4935](https://github.com/patternfly/patternfly-react/pull/4935))
+
+### Table
+- Fixed expandable cell content always being on `<td>` ([#4899](https://github.com/patternfly/patternfly-react/pull/4899))
+- Added support for single row radio select ([#4925](https://github.com/patternfly/patternfly-react/pull/4925))
+- Replaced or used exact lodash imports ([#4830](https://github.com/patternfly/patternfly-react/pull/4830))
+
+### Topology
+- Fixed exports ([#4927](https://github.com/patternfly/patternfly-react/pull/4927))
+- Fixed selector for cancelling dnd ([#4850](https://github.com/patternfly/patternfly-react/pull/4850))
+- Updated to wrap drag event handlers in actions ([#4826](https://github.com/patternfly/patternfly-react/pull/4826))
+
+### PF3
+- **Vertical nav:**
+  - Updates to not handle modified clicks on VerticalNav links ([#4747](https://github.com/patternfly/patternfly-react/pull/4747))
```

Comment:
Can we put PF3 below Other? It's a little confusing.

Reply:
yes